frontend/src/components/LineChart/LineChart.js

```diff
@@ -1,38 +1,69 @@
 import { PropTypes } from 'prop-types'
-import { useMemo, useRef } from 'react'
+import { useCallback, useMemo, useRef } from 'react'
 import { useTheme } from '@mui/system'
-import { VictoryAxis, VictoryChart, VictoryLine, VictoryScatter, VictoryTooltip } from 'victory'
+import {
+  VictoryAxis,
+  VictoryChart,
+  VictoryLine,
+  VictoryScatter,
+  VictoryTooltip,
+  VictoryVoronoiContainer,
+} from 'victory'
 import Box from '@mui/material/Box'
+import { useIntl } from 'react-intl'
 
-import { nFormatter } from 'utils/helpers'
-import { getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import { formatTickNumber, getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import { ChartAxisType } from 'utils/propTypes'
 import ChartFlyOut from 'components/ChartFlyOut'
 
 const LINE_CHART_RATIO = 0.75
 
-const LineChart = ({ data, title, xAxisLabel, yAxisLabel }) => {
+const LineChart = ({ data, title, xAxis, yAxis }) => {
+  const { formatNumber } = useIntl()
   const theme = useTheme()
   const domain = useMemo(() => getLineChartDomain(data, LINE_CHART_RATIO), [data])
   const victoryTheme = useVictoryTheme(theme)
-
   const ref = useRef()
   const size = useClientSize(ref)
 
+  const handleXTickFormat = useCallback(
+    t => {
+      return formatTickNumber(t, formatNumber, xAxis)
+    },
+    [formatNumber, xAxis],
+  )
+
+  const handleYTickFormat = useCallback(
+    t => {
+      return formatTickNumber(t, formatNumber, yAxis)
+    },
+    [formatNumber, yAxis],
+  )
```

Comment:
No way to reduce these into a single `handleTickFormat(axis)`? Or is that not how `useCallback` can work?

Reply:
That will cause prop changes every time. Better to leave it as is now.